cleaning code

Mayy45 opened the pull request.

Mayy45 committed:
Progres 14.47

Miku21750 reviewed (approved):
Cleaning by @Mayy45 approved by @Miku21750

ありがとう

Miku21750 committed:
Add comments and clarify context usage in service pages

Added explanatory comments for accessing updateDraft from DraftContext in service-materialApo.jsx and service-mo_detailApo.jsx. Also improved code readability in service-materialApo.jsx by adding comments for fetch functions and UI sections.
